README.md

```diff
@@ -42,6 +42,7 @@ This library currently supports the following cryptocurrencies and address forma
  - BNB (bech32)
  - XLM (ed25519 public key)
  - ATOM (bech32)
+ - ZIL (bech32)
```

Comment:
Can you provide reference link so that we can verify?

Reply:
@makoto  https://github.com/trustwallet/wallet-core/blob/master/tests/Zilliqa/AddressTests.cpp

Reply:
Probably this is better reference https://github.com/Zilliqa/Zilliqa/wiki/Address-Standard#recommended-implementation @Arachnid Do you want them to be in the README of this repo or would it be better to be included in the EIP?

Reply:
@makoto Both would be ideal.